tests/unit/client.bats

```diff
@@ -21,7 +21,7 @@ teardown() {
 }
 
 @test "dokku client (no args should print help)" {
-  run /bin/bash -c "./contrib/dokku_client.sh | head -1 | grep -q 'dokku COMMAND <app>'"
+  run /bin/bash -c "./contrib/dokku_client.sh | head -1 | egrep -q '^Usage: dokku \[.+\] COMMAND <app>.*'"
```

Comment:
The help message header changed. This allows for additional `dokku` args without changing the test.